pkg/config/validation/testdata/crds/authz-valid.yaml

```diff
@@ -81,7 +81,7 @@ spec:
   rules:
     - from:
         - source:
-            serviceAccounts: ["baz/sa"]
+            serviceAccounts: ["baz/sa", "sa"]
```

Comment:
I'm not really sure where this change is coming from?

Reply:
This is from istio/api - we share the same test cases between the two repos

Reply:
bin/update_crds.sh specifically